tests/mobile/core/website_status.test.js

```diff
@@ -0,0 +1,30 @@
+const {setUp, tearDown} = require('../../bootstrap');
+const {waitForWSMessage, replaceWebsocket} = require('../../_utils/websocket');
+
+let browser, context;
+
+beforeAll(async () => {
+    const out = await setUp({
```

Comment:
can we share this configuration block?

Reply:
The shared logic is done in the `setUp` method, we are just assigning `browser` and `context` here. sharing those would cause our test cases to use the same context, which might affect the test cases. I'm open for suggestions tho.

Reply:
I was referring to `userAgent` and `viewport` in [here](https://github.com/binary-com/deriv-app/pull/2043/files#diff-83e29bbfbae9825f11e06b4ec1dc612f84e81b789e4b53e816d28be8a2fdb1e4R10-R17) 😄

Reply:
I see what you mean. Yes, extracted the config to bootstrap.